apps/web-player/.build/webpack.config.cdn.js

```diff
@@ -5,7 +5,7 @@ const { output, resolve, devServer, rules, plugins } = require('@podlove/build')
 const version = require('../package').version
 const playerAssets = path.resolve('./node_modules/@podlove/player/dist')
 
-const BASE = `//cdn.podlove.org/web-player/${version}/`
```

Comment:
I think something went wrong with the rebase, this shouldn't be part of your commit